maintaining.rst

```diff
@@ -0,0 +1,276 @@
+
+If you want to see what is on the agenda, refer to the issues in the `community-topics repository <https://github.com/ansible-community/community-topics>`_. If you want to submit a topic, create an issue in the repository.
+
+.. _Committing:
+
+Committing
+==========
+
+Maintainers review and merge pull requests following the `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_, `Review checklist <review_checklist.rst>`_, and the `Committer guidelines <https://docs.ansible.com/ansible/devel/community/committer_guidelines.html#general-rules>`_.
+
+There can be two kinds of maintainers: :ref:`collection maintainers<Collection maintainers>` and :ref:`module maintainers<Module maintainers>`.
+
+For the both kinds it is worth keeping in mind that “with great power comes great responsibility”.
+
+.. _Collection maintainers:
+
+Collection maintainers
+----------------------
+
+Collection-scope maintainers are contributors who have the ``write`` or higher access level in a collection.
+
+They have the commit right and can merge pull requests among other permissions.
+
+If applicable, the collection maintainers expand a pull of module maintainers.
+
+.. _Module maintainers:
+
+Module maintainers
+------------------
+
+Module-scope maintainers exist in collections that have the `collection bot <https://github.com/ansible-community/collection_bot>`_,
+for example `community.general <https://github.com/ansible-collections/community.general>`_
+and `community.network <https://github.com/ansible-collections/community.network>`_.
+
+Being a module maintainer is the stage prior to becoming a collection maintainer.
+
+Module maintainers are contributors who are listed in ``.github/BOTMETA.yml``.
+
+The scope can be any file (for example, a module or plugin), directory, or repository.
+
+Because in most cases the scope is a module or group of modules, we call these contributors as module maintainers.
+
+The collection bot notifies module maintainers when issues / pull requests related to files they maintain are created.
+
+Module maintainers have the indirect commit right implemented through the `collection bot <https://github.com/ansible-community/collection_bot>`_.
+When two module maintainers comment with the keywords ``shipit``, ``LGTM``, or ``+1`` a pull request
+which changes a module they maintain, the collection bot will merge the pull request automatically.
+
+For more information about the collection bot and its interface,
+refer to the `Collection bot overview <https://github.com/ansible-community/collection_bot/blob/main/ISSUE_HELP.md>`_.
+
+When a collection maintainer considers a contribution to a file significant enough
+(it can be, for example, fixing a complex bug, adding a feature, providing regular reviews, and so on),
+they can offer the author to become a module maintainer, in other words, to add their GitHub login to ``.github/BOTMETA.yml``.
```

Comment:
Not important though, feel free to ignore
```suggestion
they can offer the author to become a module maintainer, in other words, to add their GitHub account to ``.github/BOTMETA.yml``.
```

Reply:
LGTM, committed, thanks!